### ISSUE TITLE
The Next Big Blue-Collar Job Is Coding | WIRED

### ISSUE THREAD

**Issue by janzeteachesit** [janzeteachesit/100-days-of-writing#1130]
The Next Big Blue-Collar Job Is Coding | WIRED http://ift.tt/2kmYpmX GRIST4&nbsp; STEM CSP&nbsp; CT&nbsp; The Next Big Blue-Collar Job Is Coding | WIRED When I ask people to picture a coder, they usually imagine someone like Mark Zuckerberg: a hoodied college dropout who builds an app in a feverish 72-hour programming jag&mdash;with the goal of getting insanely rich and, as they say, &ldquo;changing the world.&rdquo; But this Silicon Valley stereotype isn&rsquo;t even geographically accurate. The Valley employs only 8 percent of the nation&rsquo;s coders. All the other millions? They&rsquo;re more like Devon, a programmer I met who helps maintain a &shy;security-software service in Portland, Oregon. He isn&rsquo;t going to get fabulously rich, but his job is stable and rewarding: It&rsquo;s 40 hours a week, well paid, and intellectually challenging. &ldquo;My dad was a blue-&shy;collar guy,&rdquo; he tells me&mdash;and in many ways, Devon is too. Politicians routinely bemoan the loss of good blue-collar jobs. Work like that is correctly seen as a pillar of civil middle-class society. And it may yet be again. What if the next big blue-collar job category is already here&mdash;and it&rsquo;s programming? What if we regarded code not as a high-stakes, sexy affair, but the equivalent of skilled work at a Chrysler plant? Among other things, it would change training for programming jobs&mdash;and who gets encouraged to pursue them. As my friend Anil Dash, a technology thinker and entrepreneur, notes, teachers and businesses would spend less time urging kids to do expensive four-year computer-&shy;science degrees and instead introduce more code at the vocational level in high school. You could learn how to do it at a community college; midcareer folks would attend intense months-long programs like Dev Bootcamp. There&rsquo;d be less focus on the wunderkinds and more on the proletariat. These sorts of coders won&rsquo;t have the deep knowledge to craft wild new algorithms for flash trading or neural networks. Why would they need to? That level of expertise is rarely necessary at a job. But any blue-collar coder will be plenty qualified to sling Java&shy;Script for their local bank. That&rsquo;s a solidly middle-class job, and middle-class jobs are growing: The national average salary for IT jobs is about $81,000 (more than double the national average for all jobs), and the field is set to expand by 12 percent from 2014 to 2024, faster than most other occupations. Across the country, people are seizing this opportunity, particularly in states hit hardest by deindustrialization. In Kentucky, mining veteran Rusty Justice decided that code could replace coal. He cofounded Bit Source, a code shop that builds its workforce by retraining coal miners as programmers. Enthusiasm is sky high: Justice got 950 applications for his first 11 positions. Miners, it turns out, are accustomed to deep focus, team play, and working with complex engineering tech. &ldquo;Coal miners are really technology workers who get dirty,&rdquo; Justice says. Meanwhile, the Tennessee nonprofit CodeTN is trying to nudge high school kids into coding programs at community colleges. Some students (and teachers) worry that the kids don&rsquo;t fit the Zuckerbergian clich&eacute;. That&rsquo;s a cultural albatross, CodeTN cofounder Caleb Fristoe says. &ldquo;We need to get more employers saying, &lsquo;Yeah, we just need someone to manage the login page,&rsquo;&rdquo; he says. &ldquo;You don&rsquo;t have to be a superstar.&rdquo; Now, to be sure, society does need some superstars! Serious innovators, at companies and in academia, are the ones who create new fields like machine learning. But that doesn&rsquo;t preclude a new mainstream vision of what most programming work actually* is*. For decades, pop culture (and, frankly, writers like me) have overpromoted the &ldquo;lone genius&rdquo; coder. We&rsquo;ve cooed over the billionaire programmers of The Social Network and the Anonymized, emo, leather-clad hackers of Mr. Robot. But the real heroes are people who go to work every day and turn out good stuff&mdash;whether it&rsquo;s cars, coal, or code. *This article appears in the December 2016 issue. Subscribe now.<br>
The Next Big Blue-Collar Job Is Coding | WIRED<br>
Label: grist<br>
Date: September 11, 2017 at 08:21PM
